Sources/TuistGenerator/Mappers/ResourcesProjectMapper.swift

```diff
@@ -78,10 +78,9 @@ public class ResourcesProjectMapper: ProjectMapping { // swiftlint:disable:this
             sideEffects.append(sideEffect)
         }
 
-        if project.isExternal,
```

Comment:
I'm not sure why we enforced it only for external dependencies and not all targets like SPM does? Can revert that if needed.
But I think that since we removed the swift2objc accessor we should supply the macro for internal modules as well

Reply:
This will run the logic for _all_ targets but I think it should be fine. I think the reasoning was that we needed that piece for external dependencies, so we didn't want to change anything for internal targets. But yeah, should be fine imho